apps/api/src/server/applications/application/documents/__tests__/delete-document.test.js

```diff
@@ -0,0 +1,117 @@
+import test from 'ava';
+import sinon from 'sinon';
+import supertest from 'supertest';
+import { app } from '../../../../app.js';
+import { databaseConnector } from '../../../../utils/database-connector.js';
+import { DOCUMENT_STATUS } from '../../../../utils/document-enum.js';
```

Comment:
you should be able to get this from the `document.machine.js` file, right at the bottom `mapObjectKeysToStrings(documentStatesForMachine)` should auto-generate approximately this object, so would would not need to update it if you change the statuses for a document

Reply:
this doesn’t seems to be generating the published status

Reply:
Yeah, i would say that we should add it to the `documentStatesForMachine` object. Should be able to do something like:
```
const documentStatesForMachine = {
	awaiting_upload: {
		on: {
			uploading: 'awaiting_virus_check'
		}
	},
	awaiting_virus_check: {
		on: {
			check_success: 'not_user_checked',
			check_fail: 'failed_virus_check'
		}
	},
	not_user_checked: {},
	failed_virus_check: {},
        published: {}
};
```